js/index.js

```diff
@@ -0,0 +1 @@
+console.log("Hi");
```

Comment:
What's the purpose of this file with just one console log?

Reply:
Changed it in the latest one.